cms/core/wagtail_hooks.py

```diff
@@ -28,4 +29,14 @@ def editor_js() -> str:
     return format_html('<script src="{}"></script>', static("js/wagtail-editor-customisations.js"))
 
 
+@hooks.register("after_create_snippet")
+@hooks.register("after_edit_snippet")
+def set_glossary_term_updated_by_to_current_user_after_create_and_edit(request, instance):
+    """Automatically set the last updating user on Glossary Term on creation and modification."""
+    if instance(instance, GlossaryTerm):
+        instance.updated_by = request.user
+        instance.save()
```

Comment:
```suggestion
```

It should not need the save here. These hooks are called in the create/edit form's `form_valid` method, before the instance is saved

Reply:
alas, it is needed. discussed in Slack, and given we're using `RevisionMixin`, we have this data from the latest revision 🤦🏼 so we're cleaning this up